src/ansys/aedt/core/modeler/circuits/primitives_circuit.py

```diff
@@ -1262,6 +1262,57 @@ def create_wire(self, points, name=""):
         except Exception:
             return False
 
+    def create_page(self, name):
+        """Add a new circuit schematic page.
+
+        Parameters
+        ----------
+        name : str, int or float
+            Name to be used when creating the new circuit schematic.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        References
+        ----------
+        >>> oEditor.CreatePage()
+
+        Examples
+        --------
+        >>>from ansys.aedt.core import MaxwellCircuit
+        >>>app=MaxwellCircuit ()
+        >>>component=app.modeler.schematic.create_page(name=page_name)
+
+        """
+        if not isinstance(name, (str, int, float)):
+            self.logger.error("Wrong type for argument ``name``.")
+            return False
+        self.oeditor.CreatePage(name)
+        return True
+
+    def get_num_pages(self):
```

Comment:
I think it would be better to add `nb_pages` as a property instead of adding this feature as a method.